### DISCUSSION TITLE
Map Release Standardization

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#159] [Announcements]
When releasing custom content via GitHub discussions, NZ:P Team members can assign several labels to the item to categorize them. One of these labels is "Meets Install Standard". 

The idea of this label is to encourage everyone to use the same standard for packaging release files so that they are consistent, concise, preserved properly, and easy to follow.

Here are the standards in place to be assigned this label:
* The item **must** be packaged as a .ZIP archive, where the `root` of the archive matches the `nzp` game folder.
  - Meaning when the archive is opened you should be greeted immediately with the folders `maps`, `sounds`, etc.
* The item page **must** have a clear, visible "DOWNLOAD" text hyperlink (not an image!), formatted as such:
  - `# [DOWNLOAD](your_link_here)`
* The item **must** run on all supported platforms, as a single package (no FTE-specific downloads).
* The item **must** not overwrite game files like sounds or models.
* The download link must be a direct download to the item, not an adfly/url shortener (which we won't allow at all) or a Dropbox landing. Clicking the hyperlink should immediately begin the download.
* The download **must** be hosted on [archive.org](https://archive.org)!
* If your item uses `mapname.txt` for level names and descriptions, [it must be formatted to spec](https://nzportable.forumotion.com/t2257-tutorial-adding-a-description-and-thumbnail-for-your-custom-maps#18680).

Following these standards ensures we won't have mishaps with services going down, pruning files, platform migration, and potential scrapping for an alternative front-end navigation. They are not forced but items without this label often means they are too messy/too confusing.

You can check out examples of this standard here:
https://github.com/nzp-team/nzportable/discussions/categories/map-releases?discussions_q=category%3A%22Map+Releases%22+label%3A%22Meets+Install+Standard%22

Thanks!

**Comment by Paindeluxe** [comment]
> Map Release Standardization

Why not Release Standardization

Everyone would benefit from it!

Mappers and more who are looking for items in [Other Content Releases](https://github.com/nzp-team/nzportable/discussions/categories/other-content-releases) can also benefit from this!
